be/tool/beck.c

```diff
@@ -2742,20 +2764,25 @@ static void cob_act(struct action *act, struct m0_be_tx *tx)
 		adom = stob_ad_domain2ad(sdom);
 		prefix = M0_UINT128(stob_id.si_fid.f_container,
 				    stob_id.si_fid.f_key);
+
+		fid.f_container = stob_id.si_fid.f_container;
+		fid.f_key = stob_id.si_fid.f_key;
+		ht_idx = m0_stob_get_hash(&fid);
+
 		emap_dom_find(&ca->coa_act,
-			      &adom->sad_adata.em_mapping.bb_backlink.bli_fid,
+			      &adom->sad_adata_ht[ht_idx].sad_adata.em_mapping.bb_backlink.bli_fid,
```

Comment:
These lines are more than 80 chars width limit.

Reply:
Handled in commit id 5cfda07